packages/nimble-components/CONTRIBUTING.md

```diff
@@ -100,6 +100,7 @@ Create a new folder named after your component with some core files:
 | types.ts                               | Contains any enum-like types defined by the component                                                                                                                                                                                                                      |
 | models/                                | A folder containing any classes or interfaces that are part of the component API or implementation                                                                                                                                                                         |
 | components/                            | A folder containing any components that are used within the component but are not exported as public components themselves.                                                                                                                                                |
+| testing/component-name.pageobject.ts   | Page object to ease testing of this component. Not required for every component.                                                                                                                                                                                           |
```

Comment:
```suggestion
| testing/component-name.pageobject.ts   | Page object to ease testing of this component.                                                                                                                                                                                   |
```
We have established a policy that pageobjects are not required for each component. We don't have them yet, but doesn't mean we shouldn't have them.

Reply:
Removed that line